client/src/app/shared/shared-main/account/actor-avatar-info.component.ts

```diff
@@ -58,4 +68,8 @@ export class ActorAvatarInfoComponent implements OnInit {
   get avatarFormat () {
     return `${$localize`max size`}: 192*192px, ${this.maxAvatarSizeInBytes} ${$localize`extensions`}: ${this.avatarExtensions}`
   }
+
+  get hasAvatar () {
+    return Actor.GET_ACTOR_AVATAR_URL(this.actor)
```

Comment:
Please don't use a getter that returns the result of a function, it's a bad practice and could lead to detector change issues.